problems/countWords.js

```diff
@@ -14,8 +14,8 @@
 * returns: {}
 */
 
-function countWords() {
-  
+function countWords(arr) {
+  return arr.countWords
```

Comment:
Check out some of our class notes on how we make counter objects. Don't be afraid to google.